.gitmodules

```diff
@@ -31,12 +25,18 @@
 [submodule "src/lib/ecl"]
 	path = src/lib/ecl
 	url = https://github.com/PX4/ecl.git
-[submodule "cmake/cmake_hexagon"]
-	path = cmake/cmake_hexagon
-	url = https://github.com/ATLFlight/cmake_hexagon
 [submodule "src/drivers/gps/devices"]
 	path = src/drivers/gps/devices
 	url = https://github.com/PX4/GpsDrivers.git
 [submodule "src/lib/micro-CDR"]
 	path = src/lib/micro-CDR
 	url = https://github.com/eProsima/micro-CDR.git
+[submodule "platforms/nuttx/src/modules/uavcan/libuavcan"]
```

Comment:
What does board ID mean in this context? Can you point to an example so I can see how it's currently done? My assumption was it's fairly similar to the way a pixhawk (px4fmu-v2 + px4io-v2) is handled. px4fmu-v2 and px4io-v2 are two independent boards/builds, but the px4fmu-v2 build consumes the px4io-v2 binary to load at boot.